pkg/trace/fileserver.go

```diff
@@ -16,10 +16,10 @@ import (
 	"strings"
 	"time"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/credentials"
-	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/aws"             //nolint:depguard
+	"github.com/aws/aws-sdk-go/aws/credentials" //nolint:depguard
+	"github.com/aws/aws-sdk-go/aws/session"     //nolint:depguard
+	"github.com/aws/aws-sdk-go/service/s3"      //nolint:depguard
```

Comment:
FYI we disabled depguard on `main`. Ref: https://github.com/celestiaorg/celestia-core/blob/0af1e151d474646f571de2eba387dc561d1dd60d/.golangci.yml#L5

We may consider doing the same on v0.34.x-celestia b/c IMO it's cumbersome to deal with. Alternatively, we can add `"github.com/aws/aws-sdk-go/` to https://github.com/celestiaorg/celestia-core/blob/05543c8ed486a9c09d0b3100a14d34c19e7c41ec/.golangci.yml#L56-L63

Reply:
good idea, fixed in https://github.com/celestiaorg/celestia-core/pull/1290/commits/7f4d81d330af738f7752b92bbd6606710cb32b77